docs/docsite/rst/intro_inventory.rst

```diff
@@ -149,8 +151,9 @@ It is also possible to make groups of groups using the ``:children`` suffix. Jus
    southwest
    northwest
 
-If you need to store lists or hash data, or prefer to keep host and group specific variables
-separate from the inventory file, see the next section.
+If you need to store lists or hash data, or prefer to keep host and group specific variables separate from the inventory file, see the next section.
+Child groups have a couple of properties to note, first, any host that is member of a child group is automatically a member of the parent group.
```

Comment:
Suggest changing this to a bulleted list, or ending the sentence at the word "note"